templates/tutorialv2/view/content_online.html

```diff
@@ -36,16 +36,26 @@
 {% endblock %}
 
 {% block description %}
-    {{ content.description }}
+    {% if content.description %}
+        {{ content.description }}
+    {% else %}
+        Site et association de partage de connaissances animé par sa communauté. Vous y trouverez des tutoriels et des articles de tous niveaux et des forums.
+    {% endif %}
 {% endblock %}
 
 
 
 {% block meta_image %}{% spaceless %}
     {% if content.image %}
-        {{ content.image.physical.tutorial_illu.url }}
+        {{ content.image.physical.social_network.url }}
     {% else %}
-        {{ block.super }}
+        {% if content.type == 'TUTORIAL' %}
+            /static/images/tutorial-illu.png
+        {% elif content.type == 'ARTICLE' %}
+            /static/images/article-illu.png
```

Comment:
pourrais-tu ici t'inspirer de ces code: https://docs.djangoproject.com/fr/2.1/howto/static-files/#staticfiles-in-templates

Reply:
Fait :)